ui/desktop/app/components/session/proxy-url/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { TYPE_TARGET_SSH } from 'api/models/target';
+
+export default class SessionProxyUrlComponent extends Component {
+  @service intl;
+
+  @tracked selectedValue =
+    this.args.targetType === TYPE_TARGET_SSH
```

Comment:
Is this needed? Don't we get a `target` model which means we have access to the `isSSH` helper from that model?

Reply:
I originally was passing in the whole model in to this component. I personally am not a fan of passing into an entire model if we only need one property from that model. I felt it was unneeded to couple this component to the target model in that way.